### ISSUE TITLE
Where do I enter my server IP adress in the OPEN GATEWAY CONFIGURATION FILE ?

### ISSUE THREAD

**Issue by jgulysse** [trinity-project/trinity#71]
Install Trinity Routing Node Gateway
Open gateway configuration file

vi gateway/config.py
Find'cg_public_ip_port = "localhost:8089"' and Put user’s public ip address at the localhost

eg：cg_public_ip_port = "8.8.8.8:8089"

My question is do I have to replace "8.8.8.8:8089" by my server IP adress  in the command line?

**Comment by zhuofanxu** [comment]
No! you just need to replace the `localhost` part of `cg_public_ip_port`  by your server ip address in the configurate file `gateway/config.py`

**Comment by jgulysse** [comment]
Hello
Thank you for your response.

I did put my IP address replacing localhost after that do I press enter ?

Or I use the screen command to create a new session window as mentioned in
the GitHub procedure?

Thanks for your help




 After the procedure mention to use Screen command to open another session

**Comment by jgulysse** [comment]
Hey,

I just signed the petition "Mme Valérie Plante - mairesse de Montréal: LA
5G À MONTRÉAL : UN G DE TROP !" and wanted to see if you could help by
adding your name.

Our goal is to reach 25,000 signatures and we need more support. You can
read more and sign the petition here:

http://chng.it/wb8mw4JS2Z

Thanks!
Jean-Garcia
